data/json/furniture_and_terrain/terrain-triffid.json

```diff
@@ -0,0 +1,197 @@
+[
+  {
+    "type": "terrain",
+    "id": "t_triffid_bark_wall",
+    "name": "solid bark wall",
+    "description": "A solid wall of bark , enclosing the trunk of a massive tree.  It will probably burn quite well.",
+    "symbol": "LINE_OXOX",
+    "color": "brown",
+    "move_cost": 0,
+    "coverage": 100,
+    "roof": "t_barkfloor",
+    "flags": [ "FLAMMABLE", "NOITEM", "SUPPORTS_ROOF", "WALL", "NO_SCENT", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "connects_to": "WALL",
+    "bash": {
+      "str_min": 60,
+      "str_max": 180,
+      "sound": "crunch!",
+      "sound_fail": "whump!",
+      "ter_set": "t_bark_wall_chipped",
+      "items": [ { "item": "splinter", "count": [ 5, 10 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_bark_wall_chipped",
+    "name": "chipped bark wall",
+    "description": "A solid wall of bark enclosing the trunk of a massive tree.  Some of it has been broken away.",
+    "symbol": "LINE_OXOX",
+    "color": "brown",
+    "move_cost": 0,
+    "coverage": 100,
+    "flags": [ "FLAMMABLE", "NOITEM", "SUPPORTS_ROOF", "WALL", "REDUCE_SCENT", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "connects_to": "WALL",
+    "bash": {
+      "str_min": 40,
+      "str_max": 160,
+      "sound": "crunch!",
+      "sound_fail": "whump!",
+      "ter_set": "t_bark_wall_broken",
+      "items": [ { "item": "splinter", "count": [ 5, 10 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_bark_wall_broken",
+    "name": "broken bark wall",
+    "description": "A solid wall of bark, completely destroyed and waiting to collapse.",
+    "symbol": "&",
+    "color": "brown",
+    "move_cost": 0,
+    "flags": [ "TRANSPARENT", "FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "REDUCE_SCENT", "PERMEABLE", "CONNECT_TO_WALL" ],
+    "bash": {
+      "str_min": 8,
+      "str_max": 150,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_barkfloor",
+      "items": [ { "item": "splinter", "count": [ 10, 20 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_triffid_wood_wall",
+    "name": "solid wood wall",
+    "description": "A solid wall of naturally grown wood.  Very flammable",
+    "symbol": "LINE_OXOX",
+    "color": "brown",
+    "move_cost": 0,
+    "coverage": 100,
+    "roof": "t_barkfloor",
+    "flags": [ "FLAMMABLE", "NOITEM", "SUPPORTS_ROOF", "WALL", "NO_SCENT", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "connects_to": "WALL",
+    "bash": {
+      "str_min": 80,
+      "str_max": 180,
+      "sound": "crunch!",
+      "sound_fail": "whump!",
+      "ter_set": "t_barkfloor",
+      "items": [ { "item": "splinter", "count": [ 20, 25 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_barkfloor",
+    "name": "bark",
+    "description": "A solid piece of bark covers the floor here, flowing inwards.",
```

Comment:
"flowing inwards"?

Reply:
Oops; historical when this was designed as a spiral inside, let me fix that.